templates/core/terraform/deploy.sh

```diff
@@ -18,7 +18,7 @@ LOG_FILE="tmp$$.log"
   -n $TF_VAR_terraform_state_container_name \
   -k ${TRE_ID} \
   -l ${LOG_FILE} \
-  -c "terraform plan -out ${PLAN_FILE} && \
+  -c "terraform plan -out ${PLAN_FILE} && terraform state list"
   terraform apply -input=false -auto-approve ${PLAN_FILE} && \
-  terraform output -json > ../tre_output.json"
+  terraform output -json > ../tre_output.json
```

Comment:
Since this also runs outside the wrapper script, does it need to be "concatenated" the the apply command?